tests/semantics/manifest.html

```diff
@@ -323,7 +323,7 @@ <h2>constrained-bnodes-in-embedded-fail <a href="#constrained-bnodes-in-embedded
 &lt;&lt; _:x :b :c &gt;&gt; :p1 :o1.
 _:x :label "C".
 </pre>
-<p>Embedded bnode identifiers share the same scope as non-embedded ones. A bnode occuring both in embedded triples and in asserted triples must statisfy them all at the same time.</p>
+<p>Embedded bnode identifiers share the same scope as non-quoted ones. A bnode occuring both in quoted triples and in asserted triples must statisfy them all at the same time.</p>
```

Comment:
```suggestion
<p>Quoted bnode identifiers share the same scope as non-quoted ones. A bnode occurring both in quoted triples and in asserted triples must satisfy all occurrences at the same time.</p>
```

Reply:
This needs to go in manifest.ttl to be properly propagated to manifest.jsonld and manifest.html. I'll add a commit to do that.

Reply:
Done in 2bb968708436ae3aac2dbb034f513a3a80c34f80.